### PR TITLE
Donate/founders

### LINKED ISSUE #49: Donations/Founders Page
We need a page to say thank you to any donators.

### PR DESCRIPTION
**Pull request by matthewjamesr**
# Added Founders page

This page has been added to display our donators and major contributors, as well as thank all of our supporters.

The link for the donate button needs to be set when able.

**This closes #49**.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Could you include a screenshot please.

**Comment by matthewjamesr** [comment]
screen shot 2015-01-11 at 6 02 43 pm

File: _sass/libs/ionicons/_ionicons-font.scss [hunk 1]
```diff
@@ -24,4 +24,4 @@
   line-height: 1;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-}
+}
```
Comment (GrahamCampbell) [inline _sass/libs/ionicons/_ionicons-font.scss]: please revert that change

File: package.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
 {
+<<<<<<< HEAD
```
Comment (GrahamCampbell) [inline package.json]: Something's gone wrong with your rebase here.

**Comment by jbrooksuk** [comment]
CC @cachethq/owners before merging any suggested changes, please squash everything into one commit :)

File: founders.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                <h3>Our donators</h3>
+                <ul>
+                  <li>Joe Cohen</li>
+                  <li>Sam McLeod</li>
+                </ul>
+              </div>
+              <div class="col-md-6">
+                <h3>Major contributors</h3>
+                <ul>
+                  <li>jbrooksuk</li>
```
Comment (GrahamCampbell) [inline founders.html]: Maybe use people's actual names here?

Comment (jbrooksuk) [inline founders.html]: And also links to their website/GitHub profile?

**Comment by matthewjamesr** [comment]
@GrahamCampbell That was not me, that must be from gulp.

**Comment by matthewjamesr** [comment]
Too hard to know everyone's name, can link their profiles though!

**Comment by matthewjamesr** [comment]
Will push later, heading out for a bit.

**Comment by GrahamCampbell** [comment]
> CC @cachethq/owners before merging any suggested changes, please squash everything into one commit :)

Yeh, I always do if there's more than 2 commits. :)

**Comment by matthewjamesr** [comment]
Added the linkage to websites and github profiles

screen shot 2015-01-11 at 9 19 11 pm

File: _sass/libs/ionicons/_ionicons-font.scss (hunk 1 above)
Comment (joecohens) [inline _sass/libs/ionicons/_ionicons-font.scss]: I guess this change was made by ionicons on the latest version since they get copied to the libs folder.

**Comment by joecohens** [comment]
Thanks!! Not sure about the layout and typography hierarchy, but I guess this is a good starting point to work on it after squash and merge :)

**Comment by matthewjamesr** [comment]
@joecohens not a problem!! Yeah I went basic because I didn't know exactly what was in your guy's heads haha! I you would like a change etc let me know :). I am thinking of moving the github link/icons left of the names. Thoughts?

**Comment by GrahamCampbell** [comment]
Any news on this one>

**Comment by jbrooksuk** [comment]
I believe @joecohens is redesigning the site Sunday?

**Comment by joecohens** [comment]
That's right :)

**Comment by GrahamCampbell** [comment]
Awesome. :)

**Comment by matthewjamesr** [comment]
Let me know if anything is need guys :)

**Comment by jbrooksuk** [comment]
Closing this as we've now got a new website :)

**Pull request closed without merging**